src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -651,8 +651,13 @@ export class TerminalService implements ITerminalService {
 			}
 
 			if (hasParens) {
-				c(`& '${originalPath}'`);
-				return;
+				if (isPowerShell) {
+					c(`& '${originalPath}'`);
+					return;
+				} else {
+					c(`'${originalPath}'`);
+					return;
+				}
 			}
```

Comment:
Does `'path'` work on cmd and git bash or is it even needed at all? Also should this be windows specific?

Reply:
I tried it and it does work on those.

Reply:
in the issue, the person describes using both powershell and the cmd prompt, so I thought they should both work?

Reply:
interestingly, the run active file in terminal online seems to work in git bash when the path is wrapped in quotes. and currently that's not happening other than if there are parenthesis.

but the opposite is true for cmd prompt - when there are parenthesis, it doesn't work.

Reply:
> Also should this be windows specific?

can't powershell be used on non-windows devices?